101-workshop-nodejs/ex0-what-is-serverless/serverless-defined.md

```diff
@@ -38,10 +30,4 @@ _adjective_ (of computing)
 1. _Event-driven application programming using stand-alone functions with no deployment or operational considerations_
 {% endhint %}
 
-### Serverless "is a" misnomer
```

Comment:
This was a play on developer object oriented instantiation... "is a" being meaningful to programmers. OK, leaving it out as this was 1 of 3 such programmer familiar references where the other 2 I must have removed.

Reply:
Just wanted to let you know why "is a " was in quotes...

Reply:
Oh, I see! Yeah, I think it may be best to leave it out.